build.ps1

```diff
@@ -65,8 +65,7 @@ if (-not(ValidateCommand($nugetCommand))) {
     mkdir -Path "$env:LOCALAPPDATA\Nuget" -Force
     $ProgressPreference = 'SilentlyContinue'
     [Net.WebRequest]::DefaultWebProxy.Credentials = [Net.CredentialCache]::DefaultCredentials
-    Invoke-WebRequest 'https://dist.nuget.org/win-x86-commandline/latest/nuget.exe' -OutFile $nugetCommand 
-    ProcessLastExitCode $lastexitcode "Download nuget.exe"
```

Comment:
Invoke-WebRequest won't change the value of $lastexitcode even if some error happens
So if $lastexitcode default to be `not 0`, even if Invoke-WebRequest runs successfully,  an error message will be print and this script will exit.

---
In reply to: [127640655](https://github.com/dotnet/docfx/pull/1881#discussion_r127640655) [](ancestors = 127640655)